cmd/juju/commands/bootstrap.go

```diff
@@ -857,64 +827,7 @@ See `[1:] + "`juju kill-controller`" + `.`)
 		return errors.Annotate(err, "failed to bootstrap model")
 	}
 
-	agentVersion := jujuversion.Current
-	if c.AgentVersion != nil {
-		agentVersion = *c.AgentVersion
-	}
-
-	controllerDataRefresher := func() error {
```

Comment:
not need for controllerDataRefresher to be embedded. agentVersion about now included in controllerDataRefresher.